scripts/unsupported/model_segments_postprocessing_cnv/model_segments_postprocessing.wdl

```diff
@@ -51,45 +53,98 @@
 #  - centromere_tracks_seg:  gs://gatk-best-practices/somatic-hg38/final_centromere_hg38.seg
 #  - gistic_blacklist_tracks_seg:  gs://gatk-best-practices/somatic-hg38/CNV.hg38liftover.bypos.v1.CR1_event_added.mod.seg
 #
-#  Dev note:  FYI...This workflow is easily tested on a laptop.
+#  Dev notes:
```

Comment:
Add an explanation of why CNLoH events and common germline events are not distinguishable with this method.